packages/bsky/src/hydration/util.ts

```diff
@@ -25,6 +25,34 @@ export type RecordInfo<T> = {
   takedownRef: string | undefined
 }
 
+export const mergeMaps = <V, M extends HydrationMap<V>>(
+  mapA?: M,
+  mapB?: M,
+): M | undefined => {
```

Comment:
@devinivy special note: I axed the `Merges` interface here so that I could use this more easily within `mergeNestedMaps`. No change to types or functionality that I can tell, but unsure if I knocked over Chesteron's Fence anywhere else.

Reply:
Also general note that I extracted these and made a `util.test.ts` test